src/jit/codegencommon.cpp

```diff
@@ -10265,7 +10265,54 @@ void CodeGen::genCaptureFuncletPrologEpilogInfo()
 
 /*****************************************************************************
  *
- *  Generates code for an EH funclet prolog.
+ *  Generates code for an EH funclet prolog. WIP.
+ *
+ *  Funclets have the following incoming arguments:
+ *
+ *      catch/filter-handler: ebp+8 = CallerSP, ebp+12 = the exception object that was caught (see GT_CATCH_ARG)
+ *      filter:               ebp+8 = CallerSP, ebp+12 = the exception object to filter (see GT_CATCH_ARG)
+ *      finally/fault:        ebp+8 = CallerSP
+ *
+ *  Funclets set the following registers on exit:
+ *
+ *      catch/filter-handler: eax = the address at which execution should resume (see BBJ_EHCATCHRET)
+ *      filter:               eax = non-zero if the handler should handle the exception, zero otherwise (see GT_RETFILT)
+ *      finally/fault:        none
+ *
+ *  The X86 funclet prolog sequence is:
+ *
+ *     push     ebp
+ *     mov      ebp, dword ptr [esp+08H]
+ *     lea      ebp, [ebp-08H]
+ *     mov      eax, dword ptr [esp+0CH]
+ *     push     callee-saved regs   ; if necessary
+ *
+ *  The epilog sequence is then:
+ *
+ *     pop callee-saved regs        ; if necessary
+ *     pop ebp
+ *     ret
+ *
+ *  The funclet frame is thus:
+ *
+ *      |                       |
+ *      |-----------------------|
+ *      |       incoming        |
+ *      |       arguments       |
+ *      +=======================+ <---- Caller's SP
+ *      |    Return address     |
+ *      |-----------------------|
+ *      |      Saved EBP        |
+ *      |-----------------------|
+ *      |Callee saved registers |
+ *      |-----------------------|
+ *      |   Outgoing arg space  | // this only exists if the function makes a call
+ *      |-----------------------|
+ *      |       |               |
```

Comment:
It's for spaces when calling other methods, if any, inside funclets with "push" instruction.